site/docs-md/apis/browser/index.md

```diff
@@ -26,6 +26,10 @@ const { Browser } = Plugins;
 await Browser.open({ url: 'http://capacitor.ionicframework.com/' });
 ```
 
+## Note about `browserPageLoaded`
+
+In constrast to Cordova's `InAppBrowserEvent`, the `info` object does not contain a URL. You must use the [App](https://capacitor.ionicframework.com/docs/apis/app/) plugin's `appUrlOpen` to get the URL when the user navigates, or DOM events when in the browser to listen for a page load.
```

Comment:
No need to mention Cordova's `InAppBrowserEvent` here, just say that doesn't contain the URL.

Also, the part about using App plugin is not very clear, it sounds like people will be able to get any url where the Browser plugin navigates to, which is not possible. But I assume you mean navigating to an url with a custom scheme or app link that the app is configured to open, as those are registered by the `appUrlOpen`, so try to make that more clear. 
And I don't understand the part about " DOM events when in the browser to listen for a page load"